recipes/datajoint/meta.yaml~

```diff
@@ -0,0 +1,82 @@
+# Note: there are many handy hints in comments in this example -- remove them when you've finalized your recipe
```

Comment:
This file should be removed. It is probably a backup file from your text editor.